test/RdfParserTest.cpp

```diff
@@ -470,19 +489,13 @@ TEST(RdfParserTest, numericLiteralErrorBehavior) {
     return parser.parseAndReturnAllTriples();
   };
 
-  auto testTripleObjects = [&parseAllTriples]<typename T>(
-                               auto& parser, std::vector<std::string> triples,
-                               std::vector<T> expectedObjects) {
-    for (size_t i = 0; i < triples.size(); ++i) {
-      const auto& triple = triples[i];
-      std::vector<TurtleTriple> result;
-      ASSERT_NO_THROW(result = parseAllTriples(parser, triple));
-      ASSERT_EQ(result.size(), 1ul);
-      ASSERT_EQ(result[0].object_, expectedObjects[i]);
-    }
-  };
+  auto testTripleObjects =
+      TestTripleObjectsNumericLiteralErrorBehavior<decltype(parseAllTriples)>{
+          parseAllTriples};
+
```

Comment:
Can you just replace the `vector<T> ` by `auto` and keep the local lambda?

Reply:
Addressed